--- docs/gnome-help/pl/teg.sgml	2007-01-11 15:40:19.000000000 +0100
+++ docs/gnome-help/pl/teg.sgml	2007-01-11 20:32:50.000000000 +0100
@@ -1,4 +1,4 @@
-<!DOCTYPE article PUBLIC "-//GNOME//DTD DocBook PNG Variant V1.1//EN"[
+<!DOCTYPE article PUBLIC "-//GNOME//DTD DocBook PNG Variant V1.1//EN" "http://foo/bar" [
         <!-- if not using PNG graphic, replace reference above with
              .....PUBLIC "-//OASIS//DTD DocBook V3.1//EN"[
          -->
@@ -19,7 +19,7 @@
 
 <article id="index"> <!-- please do not change the id -->
   <artheader>
-  <title>PodrÍcznik gracza Tenes Empanadas Graciela</title>
+  <title>Podr—äcznik gracza Tenes Empanadas Graciela</title>
       <copyright>
         <year>2002</year>
         <holder>Quesada, Ricardo</holder>
@@ -61,23 +61,23 @@
   </artheader>
 
   <sect1 id="cintro">
-    <title>PodrÍcznik gracza TEG</title>
+    <title>Podr—äcznik gracza TEG</title>
 
     <sect2 id="introduction">
       <title>Wprowadzenie</title>
-      <para>Tenes Empandas Graciela (TEG), jest klonem argentyÒskiej gry
-      zwanej 'Plan T·ctico y Estr·tegico de la Guerra', ktÛra z kolei
-      jest zmodyfikowan± wersj± strategii turowej 'Risk'.
-      Jest to gra przeznaczona dla wielu graczy, mog±cych ≥±czyÊ siÍ za po∂rednictwem
+      <para>Tenes Empandas Graciela (TEG), jest klonem argenty—ëskiej gry
+      zwanej 'Plan T—Åctico y Estr—Åtegico de la Guerra', kt—ìra z kolei
+      jest zmodyfikowan–ë wersj–ë strategii turowej 'Risk'.
+      Jest to gra przeznaczona dla wielu graczy, mog–ëcych –ì–ëczy—Ü si—ä za po–ñrednictwem
       Internetu.
       </para>
 
       <para> 
-      Celem gry jest podbicie wszystkich krajÛw ∂wiata, wy∂wietlonych na
-      mapie lub wykonanie tajnej misji (je∂li gracze wybrali tak± opcjÍ przed
-      rozpoczÍciem gry). Do osi±gniÍcia tego celu gracze powinni obraÊ jak±∂
-      strategiÍ (np. prÛbuj±c podbiÊ jeden kontynent, broni±c granic
-      drugiego. Gdy to siÍ uda, podbiÊ kontynent, ktÛry jest rzeczywistym celem).
+      Celem gry jest podbicie wszystkich kraj—ìw –ñwiata, wy–ñwietlonych na
+      mapie lub wykonanie tajnej misji (je–ñli gracze wybrali tak–ë opcj—ä przed
+      rozpocz—äciem gry). Do osi–ëgni—äcia tego celu gracze powinni obra—Ü jak–ë–ñ
+      strategi—ä (np. pr—ìbuj–ëc podbi—Ü jeden kontynent, broni–ëc granic
+      drugiego. Gdy to si—ä uda, podbi—Ü kontynent, kt—ìry jest rzeczywistym celem).
       </para>
 
     </sect2>
@@ -86,30 +86,30 @@
   <sect1 id="crules">
     <title>Rozgrywka</title>
     <sect2 id="connecting">
-      <title>£±czenie z serwerem</title>
-      <para>Aby po≥±czyÊ siÍ z serwerem naleøy znaÊ adres oraz port serwera.
-      Dobrze znanym portem jest 2000. Naleø takøe poinformowaÊ klienta o tych
-      opcjach za po∂rednictwem okna <command>po≥±cz</command>.
+      <title>–É–ëczenie z serwerem</title>
+      <para>Aby po–ì–ëczy—Ü si—ä z serwerem nale–üy zna—Ü adres oraz port serwera.
+      Dobrze znanym portem jest 2000. Nale–ü tak–üe poinformowa—Ü klienta o tych
+      opcjach za po–ñrednictwem okna <command>po–ì–ëcz</command>.
       </para>
 
       <figure>
-      <title>Okno po≥±czenia</title>
-      <screenshot><screeninfo>okno po≥±czenia</screeninfo>
+      <title>Okno po–ì–ëczenia</title>
+      <screenshot><screeninfo>okno po–ì–ëczenia</screeninfo>
       <graphic format="png" fileref="figures/connect.png" srccredit="riq">
       </graphic></screenshot>
       </figure>
 
-      <para>Po wype≥nieniu pÛl 'Okno po≥±czenia', wy∂wietlone zostanie
-      nowe okno umoøliwiaj±ce wybÛr koloru gracza.</para>
+      <para>Po wype–ìnieniu p—ìl 'Okno po–ì–ëczenia', wy–ñwietlone zostanie
+      nowe okno umo–üliwiaj–ëce wyb—ìr koloru gracza.</para>
 
       <figure>
-      <title>Okno 'wybÛr koloru'</title>
+      <title>Okno 'wyb—ìr koloru'</title>
       <screenshot><screeninfo>okno koloru</screeninfo>
       <graphic format="png" fileref="figures/selectcolor.png" srccredit="riq">
       </graphic></screenshot>
       </figure>
 
-  <para>Je∂li zostanie wybrany kolor, ktÛry nie jest dostÍpny, serwer
+  <para>Je–ñli zostanie wybrany kolor, kt—ìry nie jest dost—äpny, serwer
   wybierze kolor za gracza.</para>
       
 
@@ -118,11 +118,11 @@
     <sect2 id="starting">
       <title>Uruchamianie gry</title>
       <para>W celu uruchomienia gry, przynajmniej 2 (maksymalnie
-      6) graczy musi byÊ przy≥±czonych do serwera. Po przy≥±czeniu, 
-      kaødy z graczy moøe zainicjowaÊ rozgrywkÍ poprzez klikniÍcie przycisku 
-      <command>uruchom</command>. Dodatkowi gracze nie mog± przy≥±czaÊ siÍ 
-       je∂li gra zosta≥a juø uruchomiona.</para>
-      <para>Po klikniÍciu przycisku <command>uruchom</command>, pojawi siÍ
+      6) graczy musi by—Ü przy–ì–ëczonych do serwera. Po przy–ì–ëczeniu, 
+      ka–üdy z graczy mo–üe zainicjowa—Ü rozgrywk—ä poprzez klikni—äcie przycisku 
+      <command>uruchom</command>. Dodatkowi gracze nie mog–ë przy–ì–ëcza—Ü si—ä 
+       je–ñli gra zosta–ìa ju–ü uruchomiona.</para>
+      <para>Po klikni—äciu przycisku <command>uruchom</command>, pojawi si—ä
       okno dialogowe 'Rodzaj rozgrywki'.</para>
 
       <figure>
@@ -132,90 +132,90 @@
       </graphic></screenshot>
       </figure>
 
-      <para>Serwer nastÍpnie rozdysponuje losowo kraje wszystkim graczom,
-      ustawiaj±c na kaødym po jednej armii.</para>
+      <para>Serwer nast—äpnie rozdysponuje losowo kraje wszystkim graczom,
+      ustawiaj–ëc na ka–üdym po jednej armii.</para>
 
       <note>
       <title>Note</title>
-      <para>Osoby mog± przy≥±czaÊ siÍ do serwera jako obserwatorzy. 
-      Umoøliwia to przygl±danie siÍ rozgrywce ale bez uczestniczenia w niej.
-      Maksymalnie moøe byÊ przy≥±czonych
-      do serwera w tym trybie 9 osÛb.</para></note>
+      <para>Osoby mog–ë przy–ì–ëcza—Ü si—ä do serwera jako obserwatorzy. 
+      Umo–üliwia to przygl–ëdanie si—ä rozgrywce ale bez uczestniczenia w niej.
+      Maksymalnie mo–üe by—Ü przy–ì–ëczonych
+      do serwera w tym trybie 9 os—ìb.</para></note>
 
     </sect2>
 
     <sect2 id="armies1">
       <title>Rozmieszczanie armii po raz pierwszy</title>
-      <para>Aby po raz pierwszy rozmie∂ciÊ armie, naleøy poczekaÊ na swoj±
-      turÍ. Je∂li gracz otrzyma turÍ musi rozmie∂ciÊ 5 armii w swoich krajach.
-      Po tej czynno∂ci naleøy wys≥aÊ armie do serwera klikaj±c przycisk
-      <command>wy∂lij armie</command>.
-      NastÍpnie naleøy poczekaÊ ponownie na swoj± turÍ, oraz rozmie∂ciÊ
-      kolejne 3 armie w swoich krajach. Moøna umieszczaÊ wszystkie armie w jednym
-      kraju (nie ma limitu na liczbÍ armii w danym kraju) lub teø w rÛønych
-      krajach. Po ustawieniu przez kaødego gracza 8 armii moøna rozpocz±Ê turÍ ataku.</para>
+      <para>Aby po raz pierwszy rozmie–ñci—Ü armie, nale–üy poczeka—Ü na swoj–ë
+      tur—ä. Je–ñli gracz otrzyma tur—ä musi rozmie–ñci—Ü 5 armii w swoich krajach.
+      Po tej czynno–ñci nale–üy wys–ìa—Ü armie do serwera klikaj–ëc przycisk
+      <command>wy–ñlij armie</command>.
+      Nast—äpnie nale–üy poczeka—Ü ponownie na swoj–ë tur—ä, oraz rozmie–ñci—Ü
+      kolejne 3 armie w swoich krajach. Mo–üna umieszcza—Ü wszystkie armie w jednym
+      kraju (nie ma limitu na liczb—ä armii w danym kraju) lub te–ü w r—ì–ünych
+      krajach. Po ustawieniu przez ka–üdego gracza 8 armii mo–üna rozpocz–ë—Ü tur—ä ataku.</para>
 
     </sect2>
 
     <sect2 id="attack">
       <title>Atakowanie przeciwnika</title>
-      <para>Aby zaatakowaÊ przeciwnika naleøy poczekaÊ na swoj± turÍ.
-     Po jej otrzymaniu moøna atakowaÊ dowoln± ilo∂Ê razy.
-     Aby zaatakowaÊ naleøy wybraÊ 2 kraje: kraj ºrÛd≥owy (gracza
-     atakuj±cego) oraz kraj docelowy (kraj przeciwnika).
-     Kraj ºrÛd≥owy musi posiadaÊ co najmniej 2 armie, lecz nie ma ograniczenia
-     na liczbÍ armii kraju docelowego. Kraje ºrÛd≥owy i docelowy musz±
-     graniczyÊ ze sob±. Krajami granicznymi s± takøe te kraje, ktÛre ≥±czy linia
+      <para>Aby zaatakowa—Ü przeciwnika nale–üy poczeka—Ü na swoj–ë tur—ä.
+     Po jej otrzymaniu mo–üna atakowa—Ü dowoln–ë ilo–ñ—Ü razy.
+     Aby zaatakowa—Ü nale–üy wybra—Ü 2 kraje: kraj –úr—ìd–ìowy (gracza
+     atakuj–ëcego) oraz kraj docelowy (kraj przeciwnika).
+     Kraj –úr—ìd–ìowy musi posiada—Ü co najmniej 2 armie, lecz nie ma ograniczenia
+     na liczb—ä armii kraju docelowego. Kraje –úr—ìd–ìowy i docelowy musz–ë
+     graniczy—Ü ze sob–ë. Krajami granicznymi s–ë tak–üe te kraje, kt—ìre –ì–ëczy linia
      na mapie.
       </para>
 
       <para>
-      W trakcie ataku gracz rzuca 'n' ko∂ci, gdzie 'n' jest liczb± 
-      armii posiadanych w kraju minus 1, lecz nie wiÍcej niø trzema ko∂Êmi.
-      Przeciwnik atakowany rzuca 'm' ko∂ci, lecz nie wiÍcej niø 3. Po rzucie 
-      wyniki s± sortowane w kolejno∂ci malej±cej.</para>
-
-      <para>Wyniki s± porÛwnywane: najwiÍkszy z najwiÍkszym, i tak dalej.
-      Je∂li jest remis lub teø wynik dla kraju docelowego jest wiÍkszy niø
-      kraju ºrÛd≥owego, z kraju ºrÛd≥owego naleøy usun±Ê jedn± armiÍ. W innym
-      przypadku armia jest usuwana z kraju docelowego.
-
-      <para>Gracz zdobywa kraj je∂li przeciwnik nie posiada juø tam armii.
-      W tym przypadku moøna przesun±Ê co najwyøej 3 armie do podbitego kraju
-      (z kraju ºrÛd≥owego)</para>
+      W trakcie ataku gracz rzuca 'n' ko–ñci, gdzie 'n' jest liczb–ë 
+      armii posiadanych w kraju minus 1, lecz nie wi—äcej ni–ü trzema ko–ñ—Ümi.
+      Przeciwnik atakowany rzuca 'm' ko–ñci, lecz nie wi—äcej ni–ü 3. Po rzucie 
+      wyniki s–ë sortowane w kolejno–ñci malej–ëcej.</para>
+
+      <para>Wyniki s–ë por—ìwnywane: najwi—äkszy z najwi—äkszym, i tak dalej.
+      Je–ñli jest remis lub te–ü wynik dla kraju docelowego jest wi—äkszy ni–ü
+      kraju –úr—ìd–ìowego, z kraju –úr—ìd–ìowego nale–üy usun–ë—Ü jedn–ë armi—ä. W innym
+      przypadku armia jest usuwana z kraju docelowego.</para>
+
+      <para>Gracz zdobywa kraj je–ñli przeciwnik nie posiada ju–ü tam armii.
+      W tym przypadku mo–üna przesun–ë—Ü co najwy–üej 3 armie do podbitego kraju
+      (z kraju –úr—ìd–ìowego)</para>
 
-      <para>Przyk≥ady rzutÛw:<para>
+      <para>Przyk–ìady rzut—ìw:</para>
       <itemizedlist>
-      <listitem><para>Gracz posiada 3 armie, jego przeciwnik takøe 3.
-      Rzuca on dwoma ko∂Êmi, za∂ przeciwnik trzema.</para></listitem>
-      <listitem><para>Je∂li gracz posiada 5 armii, za∂ przeciwnik 2,
-       to bÍdzie rzucaÊ trzema ko∂Êmi za∂ przeciwnik dwoma.</para></listitem>
-      <listitem><para>Je∂li gracz posiada jedn± armiÍ, nie moøe atakowaÊ</para></listitem>
+      <listitem><para>Gracz posiada 3 armie, jego przeciwnik tak–üe 3.
+      Rzuca on dwoma ko–ñ—Ümi, za–ñ przeciwnik trzema.</para></listitem>
+      <listitem><para>Je–ñli gracz posiada 5 armii, za–ñ przeciwnik 2,
+       to b—ädzie rzuca—Ü trzema ko–ñ—Ümi za–ñ przeciwnik dwoma.</para></listitem>
+      <listitem><para>Je–ñli gracz posiada jedn–ë armi—ä, nie mo–üe atakowa—Ü</para></listitem>
       </itemizedlist> 
 
-      <para>Przyk≥ady rzutÛw:</para>
+      <para>Przyk–ìady rzut—ìw:</para>
       <itemizedlist>
-      <listitem><para>Je∂li gracz wyrzuci: <guilabel>6, 4, 2</guilabel>,
-      za∂ przeciwnik <guilabel>6, 3</guilabel>, wynikiem jest: <guilabel>
+      <listitem><para>Je–ñli gracz wyrzuci: <guilabel>6, 4, 2</guilabel>,
+      za–ñ przeciwnik <guilabel>6, 3</guilabel>, wynikiem jest: <guilabel>
       6 przeciwko 6</guilabel>,
-      za∂ przeciwnik wygrywa (jedna z armii gracza do usuniÍcia).
-      W wyniku drugiego porÛwnania:<guilabel>4 przeciwko 3 </guilabel>, 
-      wygrywa gracz atakuj±cy (armia przeciwnika do usuniÍcia).</para></listitem>
-
-      <listitem><para>Je∂li gracz wyrzuci: <guilabel>4, 2, 2</guilabel>, 
-      za∂ przeciwnik <guilabel>3, 2, 1</guilabel>, wynikiem jest: <guilabel>
-      4 i 3</guilabel>, wygrywa gracz (przeciwnik usuwa swoj± armiÍ). 
-      PorÛwnanie drugich wynikÛw:<guilabel>2 i 2</guilabel>, 
-      wiÍc wygrywa przeciwnik (utrata 1 armii gracza).
+      za–ñ przeciwnik wygrywa (jedna z armii gracza do usuni—äcia).
+      W wyniku drugiego por—ìwnania:<guilabel>4 przeciwko 3 </guilabel>, 
+      wygrywa gracz atakuj–ëcy (armia przeciwnika do usuni—äcia).</para></listitem>
+
+      <listitem><para>Je–ñli gracz wyrzuci: <guilabel>4, 2, 2</guilabel>, 
+      za–ñ przeciwnik <guilabel>3, 2, 1</guilabel>, wynikiem jest: <guilabel>
+      4 i 3</guilabel>, wygrywa gracz (przeciwnik usuwa swoj–ë armi—ä). 
+      Por—ìwnanie drugich wynik—ìw:<guilabel>2 i 2</guilabel>, 
+      wi—äc wygrywa przeciwnik (utrata 1 armii gracza).
       Ostatni wynik to <guilabel> 2 i 1</guilabel> i wygrywa
-      gracz atakuj±cy - przeciwnik musi usun±Ê jedn± armiÍ.</para>
+      gracz atakuj–ëcy - przeciwnik musi usun–ë—Ü jedn–ë armi—ä.</para>
       </listitem>
 
       </itemizedlist> 
 
       <figure>
-      <title>Okno ko∂ci</title>
-      <screenshot><screeninfo>Okno ko∂ci</screeninfo>
+      <title>Okno ko–ñci</title>
+      <screenshot><screeninfo>Okno ko–ñci</screeninfo>
       <graphic format="png" fileref="figures.dices.png" srccredit="riq">
       </graphic></screenshot>
       </figure>
@@ -223,31 +223,31 @@
 
       <tip>
       <title>Tip</title>
-      <para>Nie atakuj kraju, ktÛry posiada wiÍcej armii niø twÛj</para>
+      <para>Nie atakuj kraju, kt—ìry posiada wi—äcej armii ni–ü tw—ìj</para>
       </tip>
 
     </sect2>
 
     <sect2 id="attack2">
-      <title>Po zakoÒczeniu ataku</title>
-      <para>Po ataku moøna <command>PrzegrupowaÊ swoje armie</command>.
-     Moøna przesun±Ê do 3 armii z jednego kraju do drugiego (musz± ze
-     sob± graniczyÊ), za∂ raz poruszona armia nie moøe byÊ przesuwana,
-     aø do kolejnej tury.</para>
-
-      <para>Po zakoÒczeniu <command>Przegrupowywania armii</command>, oraz
-      przed <command>ZakoÒczeniem tury</command> moøna <command>PobraÊ kartÍ
-      kraju</command>.
-
-      <para>Aby <command>PobraÊ kartÍ kraju</command> naleøy
-      zdobyÊ co najmniej jeden kraj w swojej turze, oraz je∂li ten warunek
-      zostanie spe≥niony, gracz otrzyma losowo wybran± kartÍ kraju.
-      W kraju reprezentowanym przez t± kartÍ (je∂li naleøy on do gracza) 
-      automatycznie zostan± dodane 2 armie. W innym przypadku bÍdzie 
-      moøna umie∂ciÊ te 2 armie po podbiciu danego kraju. Istniej± 4 rodzaje kart:
-      <command>galeon</command>, <command>dzia≥o</command>, <command>balon</command>
-      lub <command>joker</command>. Je∂li gracz ma 3 rÛøne lub teø 3 takie
-      same symbole na kartach - moøe on zamieniÊ je na dodatkowe armie.
+      <title>Po zako—ëczeniu ataku</title>
+      <para>Po ataku mo–üna <command>Przegrupowa—Ü swoje armie</command>.
+     Mo–üna przesun–ë—Ü do 3 armii z jednego kraju do drugiego (musz–ë ze
+     sob–ë graniczy—Ü), za–ñ raz poruszona armia nie mo–üe by—Ü przesuwana,
+     a–ü do kolejnej tury.</para>
+
+      <para>Po zako—ëczeniu <command>Przegrupowywania armii</command>, oraz
+      przed <command>Zako—ëczeniem tury</command> mo–üna <command>Pobra—Ü kart—ä
+	      kraju</command></para>.
+
+      <para>Aby <command>Pobra—Ü kart—ä kraju</command> nale–üy
+      zdoby—Ü co najmniej jeden kraj w swojej turze, oraz je–ñli ten warunek
+      zostanie spe–ìniony, gracz otrzyma losowo wybran–ë kart—ä kraju.
+      W kraju reprezentowanym przez t–ë kart—ä (je–ñli nale–üy on do gracza) 
+      automatycznie zostan–ë dodane 2 armie. W innym przypadku b—ädzie 
+      mo–üna umie–ñci—Ü te 2 armie po podbiciu danego kraju. Istniej–ë 4 rodzaje kart:
+      <command>galeon</command>, <command>dzia–ìo</command>, <command>balon</command>
+      lub <command>joker</command>. Je–ñli gracz ma 3 r—ì–üne lub te–ü 3 takie
+      same symbole na kartach - mo–üe on zamieni—Ü je na dodatkowe armie.
       </para>
 
       <informaltable><tgroup cols="2">
@@ -261,27 +261,27 @@
       <row><entry>...</entry><entry>...</entry></row>
       </tbody></tgroup></informaltable>
 
-      <para>Po trzeciej wymianie, naleøy podbiÊ 2 kraje
-      (zamiast jednego) aby otrzymaÊ 'kartÍ kraju'</para>
+      <para>Po trzeciej wymianie, nale–üy podbi—Ü 2 kraje
+      (zamiast jednego) aby otrzyma—Ü 'kart—ä kraju'</para>
 
-      <para>Na koÒcu trzeba nacisn±Ê przycisk <command>ZakoÒcz turÍ</command>
-      aby zakoÒczyÊ turÍ oraz przekazaÊ j± kolejnemu graczowi.</para>
+      <para>Na ko—ëcu trzeba nacisn–ë—Ü przycisk <command>Zako—ëcz tur—ä</command>
+      aby zako—ëczy—Ü tur—ä oraz przekaza—Ü j–ë kolejnemu graczowi.</para>
       
     </sect2>
 
     <sect2 id="newturn">
       <title>Nowa tura</title>
-      <para>Kiedy wszyscy gracze zakoÒcz± swoje tury, rozpoczyna siÍ nowa
-      tura. Tym razem graczem, ktÛry rozpoczyna jest gracz, ktÛry w poprzedniej
-      turze by≥ drugi. Ten gracz umieszcza 'n' armii, gdzie 'n' jest ca≥kowit± 
-      liczb± armii w swoich krajach podzielon± przez 2.
-      Je∂li podbity zosta≥ kontynent gracz otrzymuje dodatkowe armie do
-      umieszczenia: 
+      <para>Kiedy wszyscy gracze zako—ëcz–ë swoje tury, rozpoczyna si—ä nowa
+      tura. Tym razem graczem, kt—ìry rozpoczyna jest gracz, kt—ìry w poprzedniej
+      turze by–ì drugi. Ten gracz umieszcza 'n' armii, gdzie 'n' jest ca–ìkowit–ë 
+      liczb–ë armii w swoich krajach podzielon–ë przez 2.
+      Je–ñli podbity zosta–ì kontynent gracz otrzymuje dodatkowe armie do
+      umieszczenia:</para> 
       <informaltable><tgroup cols="2">
       <tbody>
       <row><entry>Kontynent</entry><entry>Liczba armii</entry> </row>
-      <row><entry>Ameryka Po≥udniowa</entry><entry>3</entry></row>
-      <row><entry>Ameryka PÛ≥nocna</entry><entry>5</entry></row>
+      <row><entry>Ameryka Po–ìudniowa</entry><entry>3</entry></row>
+      <row><entry>Ameryka P—ì–ìnocna</entry><entry>5</entry></row>
       <row><entry>Afryka</entry><entry>3</entry></row>
       <row><entry>Europa</entry><entry>5</entry></row>
       <row><entry>Azja</entry><entry>7</entry></row>
@@ -290,19 +290,19 @@
 
       <note>
       <title>Uwaga</title>
-      <para>te armie musz± byÊ rozmieszczane na podbitym kontynencie.</para> 
+      <para>te armie musz–ë by—Ü rozmieszczane na podbitym kontynencie.</para> 
       </note>
 
-      <para>Po zakoÒczeniu rozmieszczania armii przez wszystkich graczy
-       moøna atakowaÊ ponownie.</para>
+      <para>Po zako—ëczeniu rozmieszczania armii przez wszystkich graczy
+       mo–üna atakowa—Ü ponownie.</para>
       
     </sect2>
    </sect1>
 
     <sect1 id="mission">
   <title>Rozgrywka z tajnymi misjami</title>
-  <para>Istnieje moøliwo∂Ê wyoru rodzaju rozgrywki - <guilabel>tajna misja</guilabel>,
-  lub teø<guilabel>podbicie ca≥ego ∂wiata</guilabel>.</para>
+  <para>Istnieje mo–üliwo–ñ—Ü wyoru rodzaju rozgrywki - <guilabel>tajna misja</guilabel>,
+  lub te–ü<guilabel>podbicie ca–ìego –ñwiata</guilabel>.</para>
 
   <figure>
   <title>Typ gry</title>
@@ -313,28 +313,28 @@
 
   <note>
   <title>Uwaga</title>
-  <para>Je∂li wiesz øe tylko 2 graczy zamiesza siÍ zmierzyÊ, lepiej uøywaÊ
-  opcji 'podbicie ca≥ego ∂wiata'</para></note>
+  <para>Je–ñli wiesz –üe tylko 2 graczy zamiesza si—ä zmierzy—Ü, lepiej u–üywa—Ü
+  opcji 'podbicie ca–ìego –ñwiata'</para></note>
 
 
-  <para>Je∂li rozgrywka odbywa siÍ w trybie <guilabel>tajna misja</guilabel>, 
-  moøna zrealizowaÊ j± lub teø zrealizowaÊ zwyk≥± 'tajn±' misjÍ,
-  polegaj±c± na podbiciu 30 krajÛw ∂wiata. Je∂li rozgrywka odbywa siÍ w
-  trybie <guilabel>podbijania ca≥ego ∂wiata</guilabel> moøna wygraÊ tylko dziÍki
-  zdobyciu wszystkich krajÛw.</para>
+  <para>Je–ñli rozgrywka odbywa si—ä w trybie <guilabel>tajna misja</guilabel>, 
+  mo–üna zrealizowa—Ü j–ë lub te–ü zrealizowa—Ü zwyk–ì–ë 'tajn–ë' misj—ä,
+  polegaj–ëc–ë na podbiciu 30 kraj—ìw –ñwiata. Je–ñli rozgrywka odbywa si—ä w
+  trybie <guilabel>podbijania ca–ìego –ñwiata</guilabel> mo–üna wygra—Ü tylko dzi—äki
+  zdobyciu wszystkich kraj—ìw.</para>
 
-  <para>ZwyciÍzc± jest ten z graczy, ktÛry pierwszy zrealizuje swoja misjÍ.</para>
+  <para>Zwyci—äzc–ë jest ten z graczy, kt—ìry pierwszy zrealizuje swoja misj—ä.</para>
 
-  <para>Moøna takøe zmieniÊ typ gry z konsoli serwera. Po prostu naleøy
-  wpisaÊ <command>set conquer_world=0</command> lub
+  <para>Mo–üna tak–üe zmieni—Ü typ gry z konsoli serwera. Po prostu nale–üy
+  wpisa—Ü <command>set conquer_world=0</command> lub
   <command>set conquer_world=1</command>
-  aby prze≥±czyÊ miÍdzy opcjami
-  <guilabel>tajne misje</guilabel> oraz <guilabel>podbicie ∂wiata</guilabel>.</para>
+  aby prze–ì–ëczy—Ü mi—ädzy opcjami
+  <guilabel>tajne misje</guilabel> oraz <guilabel>podbicie –ñwiata</guilabel>.</para>
 
   <note>
-  <title>Ogl±danie misji</title>
-  <para>Aby zobaczyÊ swoj± tajn± misjÍ, kliknij po prostu na
-   opcjÍ menu <guilabel>podgl±d->misja</guilabel>w oknie klienta<para>
+  <title>Ogl–ëdanie misji</title>
+  <para>Aby zobaczy—Ü swoj–ë tajn–ë misj—ä, kliknij po prostu na
+	  opcj—ä menu <guilabel>podgl–ëd->misja</guilabel>w oknie klienta</para>
   </note>
   
   <figure>
@@ -350,10 +350,10 @@
     <title>Serwer</title>
     <sect2 id="staring">
       <title>Uruchamianie serwera</title>
-      <para>Moøna uruchomiÊ serwer wykonuj±c polecenie
-      <command>tegserver</command> lub teø za po∂rednictwem
-      okna klienta, klikaj±c na <guilabel>Uruchom serwer lokalnie</guilabel>
-
+      <para>Mo–üna uruchomi—Ü serwer wykonuj–ëc polecenie
+      <command>tegserver</command> lub te–ü za po–ñrednictwem
+      okna klienta, klikaj–ëc na <guilabel>Uruchom serwer lokalnie</guilabel>
+      </para>
       <figure>
       <title>Uruchamianie serwera lokalnie</title>
       <screenshot><screeninfo>Uruchamianie serwera lokalnie</screeninfo>
@@ -364,8 +364,8 @@
     </sect2>
     <sect2 id="commands">
       <title>Polecenia serwera</title>
-      <para>Serwer posiada bardzo uøyteczne polecenia do jego konfiguracji.
-      Wpisz <command>help</command>, a wy∂wietli siÍ lista pozosta≥ych poleceÒ.
+      <para>Serwer posiada bardzo u–üyteczne polecenia do jego konfiguracji.
+	      Wpisz <command>help</command>, a wy–ñwietli si—ä lista pozosta–ìych polece—ë.</para>
 
       <figure>
       <title>Polecenia serwera</title>
@@ -378,62 +378,66 @@
   </sect1>
 
   <sect1 id="crisk">
-    <title>RÛønice w stosunku do gry Ryzyko</title>
+    <title>R—ì–ünice w stosunku do gry Ryzyko</title>
     <sect2 id="riskdiff">
-      <title>RÛønice w stosunku do gry Ryzyko</title>
+      <title>R—ì–ünice w stosunku do gry Ryzyko</title>
 
-      <para>Nie znam zasad oryginalnych Ryzyka (gra≥em tylko raz lub dwa
-      baaaardzo dawno temu). Nie jestem wiÍc pewien jakie s± rÛønice
-      pomiÍdzy nimi. Je∂li je znasz, napisz mi o nich. (w jÍz. angielskim).
-      Oto rzeczy o ktÛrych poinformowowali gracze TEG:
+      <para>Nie znam zasad oryginalnych Ryzyka (gra–ìem tylko raz lub dwa
+      baaaardzo dawno temu). Nie jestem wi—äc pewien jakie s–ë r—ì–ünice
+      pomi—ädzy nimi. Je–ñli je znasz, napisz mi o nich. (w j—äz. angielskim).
+      Oto rzeczy o kt—ìrych poinformowowali gracze TEG:
       </para>
-      <table frame=all><title>TEG vs. Risk</title>
+      <table frame="all"><title>TEG vs. Risk</title>
       <tgroup cols="3">
-      <colspec colname=c1>
-      <colspec colname=c2>
-      <colspec colname=c3>
-      <spanspec spanname=hspan namest=c1 nameend=c3 align=center>
+      <colspec colname="c1">
+      <colspec colname="c2">
+      <colspec colname="c3">
+      <spanspec spanname="hspan" namest="c1" nameend="c3" align="center">
       <thead>
       <row>
-        <entry align=center>Temat</entry>
-        <entry align=center>TEG</entry>
-        <entry align=center>Ryzyko</entry>
+        <entry align="center">Temat</entry>
+        <entry align="center">TEG</entry>
+        <entry align="center">Ryzyko</entry>
       </row>
       </thead>
       <tbody>
       <row>
         <entry>Po zdobyciu kraju</entry>
-        <entry>Moøna przesun±Ê co najwyzej 3 armie do tego kraju</entry>
-        <entry>Moøna przesun±Ê ile siÍ chce armii</entry>
+        <entry>Mo–üna przesun–ë—Ü co najwyzej 3 armie do tego kraju</entry>
+        <entry>Mo–üna przesun–ë—Ü ile si—ä chce armii</entry>
       </row>
       <row>
-        <entry>Po zakoÒczeniu tury</entry>
-        <entry>Nie moøna rozmieszczaÊ armii, dopÛki nie bÍdzie siÍ mia≥o
-        øadnej karty swojego kraju, w ktÛrym moøna umie∂ciÊ tylko 2 armie</entry>
-        <entry>Moøna umie∂ciÊ 3 armie w swoim kraju</entry>
+        <entry>Po zako—ëczeniu tury</entry>
+        <entry>Nie mo–üna rozmieszcza—Ü armii, dop—ìki nie b—ädzie si—ä mia–ìo
+        –üadnej karty swojego kraju, w kt—ìrym mo–üna umie–ñci—Ü tylko 2 armie</entry>
+        <entry>Mo–üna umie–ñci—Ü 3 armie w swoim kraju</entry>
       </row>
       <row>
         <entry>Dodatkowe armie po podbiciu kontynentu</entry>
-        <entry>MUSZ° byÊ one umieszczane na tym kontynencie</entry>
-        <entry>Mog± byÊ umieszczane w dowolnym miejscu</entry>
+        <entry>MUSZ–Å by—Ü one umieszczane na tym kontynencie</entry>
+        <entry>Mog–ë by—Ü umieszczane w dowolnym miejscu</entry>
       </row>
       <row>
         <entry>Ustawianie armii</entry>
-        <entry>S± rozmieszczane na pocz±tku kaødej tury</entry>
+        <entry>S–ë rozmieszczane na pocz–ëtku ka–üdej tury</entry>
         <entry>Rozmieszczane przed atakiem</entry>
       </row>
       <row>
         <entry>Rozpoczynanie nowej rundy</entry>
-        <entry>Kaød± runde rozpoczyna inny z graczy</entry>
-        <entry>Kaød± rundÍ rozpoczyna ten sam gracz</entry>
+        <entry>Ka–üd–ë runde rozpoczyna inny z graczy</entry>
+        <entry>Ka–üd–ë rund—ä rozpoczyna ten sam gracz</entry>
       </row>
       <row>
-        <entry spanname=hspan>Je∂li znasz jeszcze jakie∂ rÛønice
-        napisz na listÍ <email>teg-list@lists.sourceforge.net</email>
-        (w jÍz. angielskim). PamiÍtaj, øe nie znam zasad gry Ryzyko!
+        <entry spanname="hspan">Je–ñli znasz jeszcze jakie–ñ r—ì–ünice
+        napisz na list—ä <email>teg-list@lists.sourceforge.net</email>
+        (w j—äz. angielskim). Pami—ätaj, –üe nie znam zasad gry Ryzyko!
         </entry>
       </row>
       </tbody>
+      </spanspec>
+      </colspec>
+      </colspec>
+      </colspec>
       </tgroup>
       </table>
     </sect2>
@@ -442,12 +446,12 @@
   <sect1 id="cbugs">
     <title>Bugs</title>
     <sect2 id="bugs">
-      <title>Zg≥aszanie b≥ÍdÛw oraz wysy≥anie ≥at w celu ulepszenia gry</title>
+      <title>Zg–ìaszanie b–ì—äd—ìw oraz wysy–ìanie –ìat w celu ulepszenia gry</title>
       
-      <para>Moøna skontaktowaÊ siÍ z autorem (w jÍzyku angielskim) pod
+      <para>Mo–üna skontaktowa—Ü si—ä z autorem (w j—äzyku angielskim) pod
       adresem:
       <email>riq@corest.com</email>
-      OgÛlne opinie dotycz±ce tej gry naleøy wysy≥aÊ na listÍ
+      Og—ìlne opinie dotycz–ëce tej gry nale–üy wysy–ìa—Ü na list—ä
       <email>teg-list@lists.sourceforge.net</email>.</para>
 
     </sect2>
@@ -458,14 +462,14 @@
   <sect1 id="chistory">
     <title>History</title>
     <sect2 id="history">
-      <title>KrÛtka historia TEG</title>
+      <title>Kr—ìtka historia TEG</title>
 
-      <para>Sebasti·n Cativa Tolosa, powiedzia≥ mi raz o tej grze. My∂lÍ,
-      øe by≥o to w 1996, w trakcie rozmowy na uniwersytecie. Tak wiÍc
-      zaczÍli∂my o niej my∂leÊ. Nie wiem z jakiego powodu projekt ten
-      umar≥ po 3 lub 4 miesi±cach. W 1998, projekt zosta≥ reaktywowany lecz
-      umar≥ ponownie po 2 tygodniach :(. W koÒcu w lutym 2000 roku rozpocz±≥em
-      tworzenie gry od nowa i wygl±da na to, øe projekt wci±ø øyje :-).   
+      <para>Sebasti—Ån Cativa Tolosa, powiedzia–ì mi raz o tej grze. My–ñl—ä,
+      –üe by–ìo to w 1996, w trakcie rozmowy na uniwersytecie. Tak wi—äc
+      zacz—äli–ñmy o niej my–ñle—Ü. Nie wiem z jakiego powodu projekt ten
+      umar–ì po 3 lub 4 miesi–ëcach. W 1998, projekt zosta–ì reaktywowany lecz
+      umar–ì ponownie po 2 tygodniach :(. W ko—ëcu w lutym 2000 roku rozpocz–ë–ìem
+      tworzenie gry od nowa i wygl–ëda na to, –üe projekt wci–ë–ü –üyje :-).   
       </para>
     </sect2>
   </sect1>

 	  	 
